src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,55 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
+import java.util.List;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
+        writeFile(readFile(fromFileName), toFileName);
+    }
+
+    private int[] readFile(String pathToFile) {
+        int[] result = new int[2];
+
+        File file = new File(pathToFile);
+        try {
+            List<String> stringList = Files.readAllLines(file.toPath());
+            for (String string : stringList) {
+                String[] buffer = string.split(",");
+                if (buffer[0].contains("supply")) {
+                    result[0] += Integer.parseInt(buffer[1]);
+                } else {
+                    result[1] += Integer.parseInt(buffer[1]);
+                }
+            }
+            return result;
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file" + pathToFile, e);
+        }
+    }
 
+    private void writeFile(int[] values, String pathToFile) {
+        StringBuilder stringBuilder = new StringBuilder();
+        stringBuilder.append("supply,").append(values[0]).append(System.lineSeparator())
+                .append("buy,").append(values[1]).append(System.lineSeparator())
+                .append("result,").append(values[0] - values[1]);
```

Comment:
you are not only writing to the file in this method but also formatting the result too. So you should divide these parts